### ISSUE TITLE
Cannot stop operator running locally and with delve enabled

### ISSUE THREAD

**Issue by JayeshThamke** [operator-framework/getting-started#75]
Dear operator-sdk team, Thanks for developing very useful tool-kit.

I have executed
`operator-sdk run --local --watch-namespace=podset-operator --enable-delve` 
it worked fine and I could debug in vscode as well. At the end I could not terminate locally running operator on terminal with `ctrl + c`. 

Without `--enable-delve` I could stop operator without terminating terminal.

Is it a bug? Or Did I try to terminate it incorrectly?

Thanks for response 
Jayesh

**Comment by camilamacedo86** [comment]
HI @JayeshThamke, 

Thank you for raise that. I think that we could add [here](https://github.com/operator-framework/operator-sdk/blob/master/cmd/operator-sdk/run/local.go#L108) the signal that is raised when we press `ctrl + c` to kill the command. 

Would you like to collab with? What do you think about to do PR with this improvement?

**Comment by JayeshThamke** [comment]
Hi @camilamacedo86  
Yes, Thanks for response and invitation. I would like to collaborate. I would like to know the _contributers' guideline and/or norms_ to create PR else I will follow general practices. If I am correct I need to create PR directly on master branch of operator sdk.
Thanks,
Jayesh

**Comment by camilamacedo86** [comment]
HI @JayeshThamke, 

It sounds wonderful. In the `contribution guideline` section into https://sdk.operatorframework.io/docs/contribution-guidelines/ you will find all docs related how to contribute with, test and etc... However, following a summary with all that I can think that you need to know to get started and help with. 

- You need to fork the repo
- Create a new branch locally and then, the PRs will be made against master with your fork/branch
- To test the changes in the tool locally you can run `make install` and then, it will install the bin made based on the code source
- To ensure that the code will pass in the lint checks you can use `make lint`

**Comment by JayeshThamke** [comment]
Hi @camilamacedo86 
I have checked the source code of local run operator-sdk, to me it is fine and tested the snippet with stubs, was all right. 
__Successful:__
Once again I tried to debug with vscode and this time _I attached the remote process (headless delve server)_ and it prompted log. This time I could also terminate the operator with SIGINT (ctrl + c). 
__What I did wrong:__
Earlier I have attached the my operator code with _vscode launch.json - (default) Launch_ configuration to operator-sdk process (with delve server) and was not really attached to it. But it is still mysterious how did I hit the breakpoints. 

So in short it is not a bug. You may close the issue. Bdw thanks for prompt support. I am still new to k8s operators and golang but from your code base I have learnt lot of stuffs.

Jayesh
